leader-election-impl/src/test/java/com/palantir/paxos/PaxosConsensusFastTest.java

```diff
@@ -183,7 +183,7 @@ public void learnerRecovery() {
         }
         PaxosLearnerImpl learner = (PaxosLearnerImpl)
                 ((DelegatingInvocationHandler) Proxy.getInvocationHandler(state.learner(0))).getDelegate();
-        PaxosStateLog<PaxosValue> log = learner.log;
+        PaxosStateLog<PaxosValue> log = learner.learnerStateLog;
```

Comment:
since you're killing `log` for everything but the actual logger, can you rename this variable too?